.buildkite/scripts/steps/code_generation/packages_codegen.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+echo --- OpenAPI Packages Code Generation
+
+(cd packages/kbn-openapi-common && yarn openapi:generate)
+check_for_changed_files "yarn openapi:generate" true
+
+(cd packages/kbn-securitysolution-lists-common && yarn openapi:generate)
+check_for_changed_files "yarn openapi:generate" true
```

Comment:
Wouldn't it yield an error on the second `cd packages/kbn-securitysolution-lists-common`, when we're already in `packages/kbn-openapi-common`? Or `check_for_changed_files` returns us back to the root repo folder?

Reply:
Parentheses do the trick. Since `cd` is used inside them it doesn't lead to changing the directory outside. It's called a [subshell](https://www.gnu.org/software/bash/manual/bash.html#Command-Grouping).

Reply:
Thank you, I missed that. Good to know.